packages/local_auth/local_auth_android/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.1
+
+* Fix getEnrolledBiometrics to be in line with iOS implementation:
+  Present biometrics that are not enrolled are no longer returned.
+
```

Comment:
If you're also going to change from face/fingerprint to strong/weak in this PR, we should include that as a changelog entry.